source/whats-new.txt

```diff
@@ -57,6 +57,77 @@ Learn what's new in:
 * :ref:`Version 3.7 <version-3.7>`
 * :ref:`Version 3.6 <version-3.6>`
 
+.. _version-6.6:
+
+What's New in 6.6
+-----------------
+
+The {+driver-short+} v6.6 release includes the following features:
+
+- Adds the ``addStage()`` method to the fluid aggregation API. You can use this method to
+  add aggregation pipeline stages individually, as shown in the following
+  example:
+
+  .. code-block:: javascript
+
+     const documents = await users.aggregate().addStage({ $project: { name: true } }).toArray();
+
+- Adds the ``cause`` and ``dependencyName`` fields to the ``MongoMissingDependencyError``
+  class. You can use these fields to programmatically determine if a package is missing
+  or why a package didn't load.
+
+- Adds the ``minRoundTripTime`` property to the ``ServerDescription`` class. This
+  property contains the minimum round-trip time over the last 10 heartbeats.
+
+- Adds the ``toJSON()`` method to the ``TopologyDescription`` class. You can use this
+  method to stringify ``TopologyDescription`` objects to JSON.
```

Comment:
Feel free to disregard but I think it is worth surfacing that while we did add this to get better debugging/info for any existing JSON.stringify usage it is still not recommended because JSON still has degraded type information. 

> We recommend using Node's util.inspect() helper to print topology descriptions because inspect properly handles all JS types and all types we use in the driver.

Any printing of any driver events or other object should be done via this ^ if users can use `util.inspect`